internal/hcs/errors.go

```diff
@@ -171,7 +171,6 @@ type SystemError struct {
 	ID     string
 	Op     string
 	Err    error
-	Extra  string
```

Comment:
Should we delete the `ExtraInfo` from `ContainerError` and `ProcessError` as well while we're at it? Not sure if they are ever used.